aws-ecr-repository/src/main/java/software/amazon/ecr/repository/Configuration.java

```diff
@@ -23,7 +23,7 @@ public JSONObject resourceSchemaJSONObject() {
         if (resourceModel.getTags() == null) {
             return null;
         } else {
-            return resourceModel.getTags().stream().collect(Collectors.toMap(tag -> tag.getKey(), tag -> tag.getValue()));
+            return resourceModel.getTags().stream().collect(Collectors.toMap(Tag::getKey, Tag::getValue, (value1, value2) -> value2));
```

Comment:
Does this affect customer behavior at all? Can we test this through UpdateHandler tests?

Reply:
I had to add a unit test for this class. This part is abstracted in the UpdateHandler tests.